packages/node_modules/@node-red/editor-client/src/js/multiplayer.js

```diff
@@ -422,6 +422,9 @@ RED.multiplayer = (function () {
                 if (location.workspace !== 0) {
                     connectInfo.location = location
                 }
+                if (localStorage.getItem("multiplayer-name") !== undefined && localStorage.getItem("multiplayer-name").length >0) {
+                    connectInfo.name = localStorage.getItem("multiplayer-name");
```

Comment:
Maybe make this `connectInfo.name = '?"+localStorage.getItem("multiplayer-name");`